source/blog/2017-07-12-unprivileged-containers-with-bwrap-oci-and-bubblewrap.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Unprivileged containers with bwrap-oci and bubblewrap
+author: giuseppe
+date: 2017-07-12 13:00 UTC
+tags: bwrap-oci, usernamespaces, bubblewrap, unprivileged, containers
+published: true
+comments: true
+---
+The introduction of user namespaces in the Linux kernel has opened the
+doors to running containers as not privileged users.
```

Comment:
I think "unprivileged" reads more nicely than "not privileged".  If you want to be clearer, I often talk about "default user logins via e.g. ssh or desktop".